README.md

```diff
@@ -1,43 +1,34 @@
-redux-watch
-===========
+# redux-watch
```

Comment:
I prefer `#` because it's more clear than `====` (for me). With `=/-` you have only two levels and if you need more you will use `####`, then why don't use `#` instead `=/-`?